docs/PUBLISHING.md

```diff
@@ -75,7 +75,7 @@ Make sure you set the following environment variables:
 Run the following command:
 
 ```sh
-make publish-bintray-debian
+make publish-bintray-debian RELEASE_TYPE=<production|development>
```

Comment:
Better to do `RELEASE_TYPE=<production|development> make publish-bintray-debian` ?
(emphasis that it's an env-var)